### DISCUSSION TITLE
Scummvm - Open Dingus Support

### DISCUSSION THREAD

**Discussion by SuperBadger77** [eduardofilo/RG350_adam_image#290] [Ideas & Requests]
Hi,

I know Adam has ScummVm support but was just watching a youtube video: https://www.youtube.com/watch?v=bfLKAcPrETU and in it they mentioned that the latest version of ScummVm has Open Dinugs support. Not sure if this is already supported but thought I'd just raise it here. If you check the change log: https://downloads.scummvm.org/frs/scummvm/2.6.0/ReleaseNotes.html it states: Added OpenDingux beta port.

So just thought I'd share this.

**Comment by eduardofilo** [comment]
Yes, yes, I have it on the checklist for the next release. I constantly have three browser tabs open to keep track of ODbeta, RA and ScummVM updates.
Actually the version of ScummVM in v2.1 already belongs to that project, it is just the previous version, that is 2.5.1.
Thanks for the notice anyway.
